connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -224,9 +219,10 @@ private boolean initialize(OperatorDiscoveryAPI.OperatorDiscoveryResult odResult
                             getClientId(),
                             getRedirectUri()));
             isInitialized = true;
+            return true;
         } catch (RuntimeException ignored) {
         }
```

Comment:
(I mean the catch block, having problems with outlining the right line with GitHub :))